src/openclsim/plot/vessel_planning.py

```diff
@@ -173,16 +175,28 @@ def add_layout_gantt_chart(traces, xmin, xmax, static):
     layout = go.Layout(
         title="GANTT Chart",
         hovermode="closest",
-        legend=dict(x=0, y=-0.2, orientation="h"),
-        xaxis=dict(
-            title="Time",
-            titlefont=dict(family="Courier New, monospace", size=18, color="#7f7f7f"),
-            range=[xmin, xmax],
-        ),
-        yaxis=dict(
```

Comment:
Soms is gebruik van dict prima, zoals hier. 
Je kan sinds kort sonarqube meldingen uitzetten via code. Via interface moet je een heel profiel aanmaken. 
Je zou nu iets kunnen doen als `# noqa: error label dict used as intended`

https://community.sonarsource.com/t/python-issue-suppression-improvements-nosonar-and-new-rules/145017